web/src/timerender.ts

```diff
@@ -423,6 +423,12 @@ export function get_full_time(timestamp: number): string {
     return formatISO(timestamp * 1000);
 }
 
+function get_current_time_to_hour(): Date {
+    const timestamp = new Date();
+    timestamp.setMinutes(0, 0);
+    return timestamp;
+}
```

Comment:
`timerender: Extract get_current_time_to_hour.` is a good commit message for this.